Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml

```diff
@@ -20,10 +21,592 @@ configuration:
+      required: false
+      isArray: false
+    - auto: PREDEFINED
+      name: include_child_org_units
+      description: Whether to return devices from all child organizational units. If this is set to true, 'org_unit_path' must be provided.
+      required: false
+      isArray: false
+      predefined:
+      - yes
+      - no
+    - name: limit
+      description: The maximum number of records to return from the collection. Default is 50.
+      required: false
+      isArray: false
+    - name: page
+      description: The page number.
+      required: false
+      isArray: false
+    - name: page_size
+      description: The number of requested results per page. Default is 50. Maximum allowed value is 100.
+      required: false
+      isArray: false
+    description: Retrieves a paginated list of company-owned ChromeOS devices.
+    name: gsuite-chromeosdevice-list
+    outputs:
+    - contextPath: GSuite.ChromeOSDevices.PageToken.NextPageToken
+      description: The token of the next page.
+      type: String
+    - contextPath: GSuite.ChromeOSDevices.ChromeOSDevices
+      description: A list of ChromeOS device objects.
+      type: List
+    - contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.deviceId
+      description: The unique ID of the ChromeOS device.
+      type: String
+    - contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.serialNumber
+      description: The ChromeOS device serial number entered when the device was enabled.
+      type: String
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.status
+      description: Status of the ChromeOS device.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.lastSync
+      description: The date and time the ChromeOS device was last synchronized with the policy settings in the Admin console.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.supportEndDate
+      description: The final date the ChromeOS device will be supported.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.annotatedUser
+      description: The user of the ChromeOS device as noted by the administrator.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.annotatedLocation
+      description: The address or location of the ChromeOS device as noted by the administrator.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.notes
+      description: Notes about this ChromeOS device added by the administrator.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.model
+      description: The ChromeOS device's model information.
+    - type: String
+      contextPath: GSuite.ChromeOSDevices.ChromeOSDevices.meid
+      description: The Mobile Equipment Identifier (MEID) or the International Mobile Equipment Identity (IMEI) for the 3G mobile card in a mobile device.
```

Comment:
Should this be:
The Mobile Equipment Identifier (MEID) or the International Mobile Equipment Identity (IMEI) for the 3G mobile card in a **ChromeOS** device.

Reply:
Correct, my mistake.